src/treeval/scripts/SummaryStats.py

```diff
@@ -209,6 +214,7 @@ def main(args):
         # graph_process_vs_peak(total_value_df)
```

Comment:
Remove lines 208-217 and replace with, this just uncomments everything and adds in some print statements so we can see what is happening.
```   
    if args.graphs:
        for x, y in regression_args.items():
            print(f"GENERATE LINEAR GRAPHS WITH: {x} -- {y}")
            graph_linear_regressions(y, tv_data)
        
        print(f"GENERATE PROCESS VS PEAK")
        graph_process_vs_peak(total_value_df)

        print(f"GENERATE PROCESS VS PEAK - LOG")
        graph_process_vs_peak_log(total_value_df)

        print(f"GENERATE GRAPH PER WORKFLOW")
        graph_per_workflow(total_value_df, workflow_names, "./")
        
        print(f"GENERATE PEAK VS CLADE")
        graph_peak_vs_clade(total_value_df)
        
        print(f"GENERATE PROCESS VS CPU")
        graph_cpu_vs_process(total_value_df)
```